src/common/arrow/src/arrow/bitmap/utils/mod.rs

```diff
@@ -68,6 +68,7 @@ pub fn set(byte: u8, i: usize, value: bool) -> u8 {
 /// panics if `i >= data.len() / 8`
```

Comment:
Can you clarify what the inconsistency is between the documentation and implementation ?